<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsAdd</class>
 <widget class="QWidget" name="ToolsAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools and Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ToolsCloseButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="ToolTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Columns">
          <property name="minimumSize">
           <size>
            <width>565</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>565</width>
            <height>360</height>
           </size>
          </property>
          <attribute name="title">
           <string>Columns</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="CloumnName">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Column Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="ColumnNumber">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Column Number</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="CustomerName">
             <property name="text">
              <string>Customer Name :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="CustomerNameEdit">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TransactionCmnt">
             <property name="text">
              <string>Transaction Comment :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="TransactionCmntEdit">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="AccNum">
             <property name="text">
              <string>Account Number :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="AccNumEdit">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="CreditAmount">
             <property name="text">
              <string>Credit Amount :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="CreditAmountEdit">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="DebitAmount">
             <property name="text">
              <string>Debit Amount :</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="DebitAmountEdit">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="SheetNames">
          <attribute name="title">
           <string>Sheet Names</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="sheetNamesLabel">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Sheet Names&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sheetNamesDesc">
             <property name="text">
              <string>Enter coma ( ',' ) separated sheetnames in order of your excel files.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPlainTextEdit" name="sheets"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Companies">
          <attribute name="title">
           <string>Companies</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="companiesLabel">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Company Related Terms&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="companiesDesc">
             <property name="text">
              <string>Enter coma ( ',' ) separated keywords which are company related.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPlainTextEdit" name="companies_keywords"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Surnames">
          <attribute name="title">
           <string>Surnames</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="surnamesLabel">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Surnames&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="surnamesDesc">
             <property name="text">
              <string>Enter coma ( ',' ) separated surnames.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPlainTextEdit" name="surnames_new"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Junk">
          <attribute name="title">
           <string>Junk</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="junkLabel">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Junk Words&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="junkDesc">
             <property name="text">
              <string>Enter coma ( ',' ) separated keywords you want to treat as junk, occuring</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="junkDesc2">
             <property name="text">
              <string>before individual names and customer names in transaction comments.</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPlainTextEdit" name="junkWords"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="SpellCorrect">
          <attribute name="title">
           <string>Spell Correct</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="words">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Words&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="wordsDesc">
             <property name="text">
              <string>Enter coma ( ',' ) separated keywords you want to spell correct.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPlainTextEdit" name="WordsNew"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ToolsCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>ToolsAdd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
